ports/servoshell/app.rs

```diff
@@ -67,6 +67,8 @@ impl App {
 
         // Implements window methods, used by compositor.
         let window = if opts::get().headless {
+            // GL video rendering is not supported on headless windows.
+            set_pref!(media.glvideo.enabled, false);
```

Comment:
Does this mean currently we have GL video rendering enabled for headless windows?

Reply:
Nope, this is disabled, but only implicitly because the previous `WindowMethod` trait method implementations for headless windows did not return information useful for GL rendering.